.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,16 @@
+name: AutoAssignReviewer
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  auto-assign-reviewer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run assignment of reviewer team
+        uses: nikosmoum/auto-assign-reviewer-team@v0.5
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` was the example in the README file. You should replace that with whatever the name of the secret you created for the subman repo is